### PR TITLE
Couple of changes to Wiki-Become-A-Mod.md and Wiki-Labels-Guide.md

### LINKED ISSUE #866: Small update on article on labels in main repo
The `easy` label should be changed to `first timers only` as we recently changed it.

https://github.com/FreeCodeCamp/wiki/blob/master/Select-Issues-for-Contributing-Using-Labels.md

image

### PR DESCRIPTION
**Pull request by theflametrooper**
Still to change:

- [x] add the "Study Group" label
- [x] fix title attribute
- [x] SEO is missing from the labels.
- [x] Add two trailing spaces at EOL
- [x] Fix conflicts and squash commits appropriately

Closes #866

### CODE REVIEW

**Comment by Rafase282** [comment]
# LGTM

File: Wiki-Labels-Guide.md [hunk 1]
```diff
@@ -38,6 +40,14 @@ Labels | Description | Use in Issue | Use in PR
  This is an *issue only* label.  
  Label an issue as **`help wanted`** if others need to work on the issue. This is usually used for new article addition help.
 
+#### Label: `easy`
+ This is an *issue only* label.
```
Comment (abhisekp) [inline Wiki-Labels-Guide.md]: Add two trailing spaces after line end

File: Wiki-Labels-Guide.md [hunk 2]
```diff
@@ -38,6 +40,14 @@ Labels | Description | Use in Issue | Use in PR
...
+ Add this label if an issue is easy to resolve and doesn't require much coding knowledge.
+
+#### Label: `first-timers-only`
+ This is an *issue only* label.
```
Comment (abhisekp) [inline Wiki-Labels-Guide.md]: Add two trailing spaces after line end

**Comment by Rafase282** [comment]
I added the issue, you forgot about it.

**Comment by theflametrooper** [comment]
@Rafase282 Sorry, didn't know #866 existed!

**Comment by Rafase282** [comment]
Is alright.

**Comment by abhisekp** [comment]
@theflametrooper please merge my PR on your PR branch https://github.com/theflametrooper/wiki/pull/1

**Comment by abhisekp** [comment]
@theflametrooper after you merge my PR, please squash it locally to keep important commits and then @Rafase282 can review our changes and merge this PR.

**Comment by theflametrooper** [comment]
@abhisekp This will be updated in a couple of hours, when I'm back from school. :+1:

File: Wiki-Labels-Guide.md [hunk 3]
```diff
@@ -38,6 +43,15 @@ Labels | Description | Use in Issue | Use in PR
  This is an *issue only* label.  
  Label an issue as **`help wanted`** if others need to work on the issue. This is usually used for new article addition help.
 
+#### Label: `easy`
+ This is an *issue only* label.  
+ This label is used for easy to resolve issues.
+
+#### Label: `first-timers-only`
+ This is an *issue only* label.  
+ This label is used to denote that the issue should only be contributed by campers without any prior experience with contributions.
```
Comment (abhisekp) [inline Wiki-Labels-Guide.md]: Add two trailing spaces after this line

File: Wiki-Labels-Guide.md [hunk 4]
```diff
@@ -68,35 +82,36 @@ Labels | Description | Use in Issue | Use in PR
...
 #### Label: `bug`
  This is an *issue only* label.  
  Assign this label to issues related to bugs in [**Wiki**](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki) or in [**FreeCodeCamp**](https://www.freecodecamp.com) or otherwise related.
 
 #### Label: `blocked`
  This is a *PR only* label.  
- Use it when the PR would be bad if merged, e.g. information that should not be there. Use blocked when the guidelines are not followed and yet it seems like others were giving it "okay to merge" or if the person has not implemented the suggested changes.  
- Assign this label when you want to let other maintainers know not to merge until further noticed. 
+ This label is used when the PR would be bad if merged, e.g. information that should not be there. This is also used when the guidelines are not followed and yet it seems like others were giving it "*okay to merge*" or if the person has not implemented the suggested changes.  
+ Assign this label when you want to let other maintainers know not to merge until further noticed.
```
Comment (abhisekp) [inline Wiki-Labels-Guide.md]: Add two trailing spaces after this line

**Comment by Rafase282** [comment]
SEO and Study Group's CampSites are missing on the labels.

**Comment by theflametrooper** [comment]
@Rafase282 Should now be ready! Don't squash :shipit: :wink:

File: Wiki-Labels-Guide.md [hunk 5]
```diff
@@ -12,19 +14,23 @@ See also: :whale: [**Label Overview**](#label-overview) **|** :dolphin: [**Label
...
+[![**Label: in progress**](https://img.shields.io/badge/Label-in_progress-3ac569.svg?style=flat-square&maxAge=2592000 "Label: in progress")](https://github.com/FreeCodeCamp/wiki/labels/in progress) | The author's WIP :construction: article(s) | :x: | :white_check_mark:
+[![**Label: Translation**](https://img.shields.io/badge/Label-Translation-0052cc.svg?style=flat-square&maxAge=2592000 "Label: Translation")](https://github.com/FreeCodeCamp/wiki/labels/Translation) | Translation :speech_balloon: related article(s) | :white_check_mark: | :white_check_mark:
+[![**Label: Study Groups %28Campsites%29**](https://img.shields.io/badge/Label-Study%20Groups%20%28Campsites%29-722872.svg?style=flat-square "Label: Study Groups (Campsites)")](https://github.com/FreeCodeCamp/wiki/labels/Study Groups %28Campsites%29) | Campsites related :tent: | :white_check_mark: | :white_check_mark:
+[![**Label: SEO**](https://img.shields.io/badge/Label-SEO-c5def5.svg?style=flat-square "Label: SEO")](https://github.com/FreeCodeCamp/wiki/labels/SEO) | Search Engine Optimization :muscle: | :white_check_mark: | :white_check_mark:
```
Comment (abhisekp) [inline Wiki-Labels-Guide.md]: `SEO` label is invisible :see_no_evil:

File: Wiki-Labels-Guide.md [hunk 6]
```diff
@@ -12,19 +14,23 @@ See also: :whale: [**Label Overview**](#label-overview) **|** :dolphin: [**Label
...
-[**![Label: duplicate](https://img.shields.io/badge/Label-duplicate-D499B9.svg?style=flat-square&maxAge=2592000 "Label: duplicate")**](https://github.com/FreeCodeCamp/wiki/labels/duplicate) | Similar or Duplicate of each other :busts_in_silhouette: | :white_check_mark: | :white_check_mark: 
+[![**Label: QA**](https://img.shields.io/badge/Label-QA-0052cc.svg?style=flat-square&maxAge=2592000 "Label: QA")](https://github.com/FreeCodeCamp/wiki/labels/QA) | Content Review :book: | :x: | :white_check_mark:
+[![**Label: help wanted**](https://img.shields.io/badge/Label-easy-AACD6E.svg?style=flat-square "Label: help wanted")](https://github.com/FreeCodeCamp/wiki/labels/help wanted) | Need help :heavy_plus_sign: in creating the article(s) | :white_check_mark: | :x:
+[![**Label: easy**](https://img.shields.io/badge/Label-help_wanted-159818.svg?style=flat-square&maxAge=2592000 "Label: easy")](https://github.com/FreeCodeCamp/wiki/labels/easy) | Help needed to resolve :heavy_plus_sign: | :white_check_mark: | :x:
```
Comment (abhisekp) [inline Wiki-Labels-Guide.md]: `easy` label is almost invisible too (not so much) :see_no_evil:

Comment (abhisekp) [inline Wiki-Labels-Guide.md]: `help wanted` and `easy` labels are mixed up

**Comment by abhisekp** [comment]
I'll make the required changes :+1:

**Pull request merged**